README.md

```diff
@@ -4,7 +4,7 @@ This package provides elements that can be used to forward packets to multiple o
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed 
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
```

Comment:
@bblaszkow06 Shall this package be published? If so please change this line to `The package can be installed...`

Reply:
Yes, I think so, our community demands it :D